### ISSUE TITLE
Compound Injector can't be reloaded

### ISSUE THREAD

**Issue by Derpford** [MeatWheeze/NeepMeat#15]
The Compound Injector says that you can "Click with an Integration Fluid Cartridge in inventory to reload". However, it doesn't do this, and having looked at the class file for it, it doesn't look like this behavior's implemented at all.

**Comment by MeatWheeze** [comment]
Ah, the tooltip text is too ambiguous. You have to click on the Compound Injector with an cartridge _while_ in your inventory to reload it.

The behaviour is here: https://github.com/MeatWheeze/NeepMeat/blob/7cc902c38d14114c1097296dbc3ee541d54e85fc/src/main/java/com/neep/neepmeat/item/CompoundInjectorItem.java#L90

**Comment by Derpford** [comment]
That'd explain it. Clicking on the Compound Injector with an integration fluid cartridge in inventory refills it. I'm not super familiar with java or fabric and haven't touched minecraft modding in a while, so I missed onClicked while skimming the file :P

I'm not sure how to disambiguate the tooltip without making it too wordy. Maybe "With inventory open: Right-click *on* this item with an Integration Fluid Cartridge to reload"?
